src/test/java/org/assertj/core/api/duration/DurationAssert_hasDays_Test.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2019 the original author or authors.
+ */
+package org.assertj.core.api.duration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.error.ShouldHaveDuration.shouldHaveDays;
+import static org.assertj.core.util.AssertionsUtil.assertThatAssertionErrorIsThrownBy;
+import static org.assertj.core.util.FailureMessages.actualIsNull;
+
+import java.time.Duration;
+
+import org.assertj.core.api.BaseTest;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+/**
+ * @author Filip Hrisafov
+ */
+@DisplayName("DurationAssert hasDays")
+class DurationAssert_hasDays_Test extends BaseTest {
+
+  @Test
+  void should_pass_if_duration_has_matching_days() {
+    assertThat(Duration.ofDays(4L)).hasDays(4L);
```

Comment:
Just a matter of style, we are converting more and more tests to the GIVEN/WHEN/THEN structure, it would be great to use it for newly made tests

Reply:
Done, let me know if there are some other tests that do not follow the new structure